phpunit.xml.dist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         backupGlobals="false"
```

Comment:
@sergiors, IMO, only `bootstrap` and this one are enough.

I think you can let the rest as default with no problems but, differently from the comment about the `colors` attribute, this is just an opinion.